client/blocks/reader-full-post/_content.scss

```diff
@@ -442,3 +442,70 @@
 	padding-left: 16px;
 	padding-right: 16px;
 }
+
+// Writing Prompts Block Styles
+.reader-full-post__story-content .jetpack-blogging-prompt {
+	border: 1px solid var(--color-neutral-5);
+	border-radius: 2px;
+	box-sizing: border-box;
+	margin-bottom: 24px;
+	padding: 24px;
+
+	.jetpack-blogging-prompt__answers {
+		display: flex;
+
+		.image-wrapper {
```

Comment:
The answer avatars are still not looking better on mobile, even after the latest changes here.

I wonder if something like this could work for both web and mobile clients, given the HTML:
```scss
.jetpack-blogging-prompt__answers {
    display: flex;

    .image-wrapper {
        height: 24px;
        width: 24px;
    }

    .image-wrapper,
    img {
        &:first-child {
            margin-left: -2px;
        }

        &:not(:first-child) {
            margin-left: -12px;
        }
    }

    .image-wrapper img,
    img {
        border: 2px solid #fff;
        border-radius: 50%;
        height: 24px;
        width: 24px;
    }
}
```

I have just some basic CSS knowledge so I don't know if I am oversimplifying things 😅 so feel free to suggest something else.

Reply:
I just noticed that though the CSS generated by the code above causes the image to have the correct size, they still do not overlap as expected. Removing `display: flex` from `.jetpack-blogging-prompt__answers` made them work properly on Android, but not on iOS 🙄 so I am guessing some WebView / browser differences between the systems.

I am thinking it might not be worth trying to fix the answers section for the mobile clients right now and maybe just hide it for them at the moment, wdyt? cc @develric @wargcm 

For hiding it on both clients without app-side changes I was thinking we could add something like this to the [reader-mobile.scss file](https://github.com/Automattic/wp-calypso/blob/f47c6faf675d8afc4d7eb8cbadb9768f2158bbc9/client/assets/stylesheets/reader-mobile.scss), does that make sense?
```css
.jetpack-blogging-prompt__answers {
    display: none;
}
```

Reply:
I'm expecting _most_ mobile clients will only see posts with that section when viewing the prompt responses already. So I don't see a problem with hiding it if it takes a ton of work trying to get it to appear correctly.

Reply:
> ```
> <div class="wp-block-jetpack-blogging-prompt jetpack-blogging-prompt">
>     <div class="jetpack-blogging-prompt__label">
>         Daily writing prompt
>     </div>
>     <div class="jetpack-blogging-prompt__text">
>         Describe a random encounter with a stranger that stuck out positively to you.
>     </div>
>     <div class="jetpack-blogging-prompt__answers">
>         <img class="jetpack-blogging-prompt__answers-gravatar" aria-hidden="true" src="redacted-url">
>         <img class="jetpack-blogging-prompt__answers-gravatar" aria-hidden="true" src="redacted-url">
>         <img class="jetpack-blogging-prompt__answers-gravatar" aria-hidden="true" src="redacted-url">
>         <a class="jetpack-blogging-prompt__answers-link" href="https://wordpress.com/tag/dailyprompt-1912" target="_blank" rel="external noreferrer noopener">View all responses</a>
>     </div>
> </div>
> ```

@thomashorta I made some updates in https://github.com/Automattic/wp-calypso/pull/76092/commits/93fb8d4a294df38bab7e670b66047fd029f744f6 based on the code you provided above.  I pasted your markup into my local env and here's what I'm seeing now (Notice `.image-wrapper` is missing  in the markup):

<img width="1403" alt="CleanShot 2023-04-24 at 16 32 38@2x" src="https://user-images.githubusercontent.com/5634774/234110129-376c36a9-1516-4ae4-857a-b56f220562dc.png">

Reply:
On Android this is looking nice! But on iOS the avatars do not overlap (though I don't think that it's looking bad, to be honest).

Android | iOS
--- | ---
<img src="https://user-images.githubusercontent.com/5091503/234129218-e17c2e97-3a1d-4a91-88ab-22fce83027a0.png" height="640"> | <img src="https://user-images.githubusercontent.com/5091503/234129307-0897a2ca-da24-4505-ae32-78aa48584a92.png" height="640">